### DISCUSSION TITLE
move cursor to another display, when the display focus changed.

### DISCUSSION THREAD

**Discussion by muhajirdev** [rxhanson/Rectangle#794] [Ideas]
Hey, thanks for developing this app. It's been super super super helpful.

I have an idea.

- sometime I am switching focus between display (using cmd + tab) or by typing the app name in spotlight.
- and I found that I need to move my mouse cursor from one display to antoher display.

It would be awesome if rectangle can detect which display is currently in focus and move the mouse cursor there. 

I know that rectangle already support moving cursor when moving the app between, and it's already an awesome starting point. 
If we can somehow also move mouse when the display focus changes, it will be even more awesome

**Comment by rxhanson** [comment]
From a developer perspective, there should be everything in Rectangle to get started on this. I don't quite have the bandwidth to do it at this time, but I'm open to merging a good pull request for it if you would like to give it a go.

**Reply by juil** [reply]
Since Rectangle already has a setting to have the mouse follow windows when they get moved to another display, would it be possible to extend that functionality?
